code/_helpers/maths.dm

```diff
@@ -134,3 +134,6 @@
 
 /matrix/proc/get_angle()
 	return Atan2(b,a)
+
+/proc/Modulus(x, y)
```

Comment:
Could do this as
````
#define MODULUS_FLOAT(X, Y) ( (X) - (Y) * round((X) / (Y)) )
````
as an optimization/to make it more clear why it's a thing when `%` exists.